.github/workflows/pre-release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         run: npm install
       - name: Build
         run: |
-          node utils/generate_version_file.js
+          ts-node utils/generate_sources.ts
```

Comment:
would it be better to run `npm run build` here?

Reply:
or at least `npm run generate:sources`

Reply:
We just need the `version.js` file that is generated